addons/dialog/functions/fnc_gui_edit.sqf

```diff
@@ -19,11 +19,20 @@
 #include "script_component.hpp"
```

Comment:
You should consider putting the include before the documentation like in CBA/ACE. Otherwise it causes your line numbers to be messed up in case of errors.

Reply:
Yeah, need to make that change project wide.

Reply:
To not add all the changes in this PR, I guess you could merge this one and then make a new PR to change it.

Reply:
yeah, but could atleast make sure that future PR's are already doing that.